config/app.default.php

```diff
@@ -427,13 +427,61 @@
      *      before being "activated"?
      */
     'Signup' => [
-//        'requireActivation' => true,
+        // 'requireActivation' => true,
     ],
 
+    /**
+     * Filesystem configuration.
+     *
+     * Several filesystem configurations allow you to manage your files
+     * separately, and each configuration will behave as an isolate mount-point.
+     */
     'Filesystem' => [
         'default' => [
             'className' => 'BEdita/Core.Local',
             'path' => WWW_ROOT . DS . '_files',
+            'url' => env('FILESYSTEM_DEFAULT_URL', null),
+        ],
+        'thumbnails' => [
+            'className' => 'BEdita/Core.Local',
+            'path' => WWW_ROOT . DS . '_files' . DS . 'thumbs',
```

Comment:
[minor] this should be `WWW_ROOT . '_files' . DS . 'thumbs'`, also `default.path` should be changed accrodingly